db/migrations/20190709160237_create_flip_storage_tables.sql

```diff
@@ -129,6 +129,9 @@ CREATE TABLE maker.flip_vat
     UNIQUE (header_id, address_id, vat)
 );
 
+CREATE INDEX flip_vat_address_id_index
```

Comment:
seems like sometimes above we're calling this same relationship an `address_index` - would be down to choose either that or `address_id_index` and use it everywhere